src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/OffenderService.kt

```diff
@@ -24,6 +27,8 @@ class OffenderService(
     const val EXACT_MATCH_MESSAGE = "Exact delius match found - adding offender to person record"
     const val PARTIAL_MATCH_MESSAGE = "Partial Delius match found"
     const val MULTIPLE_MATCHES_MESSAGE = "Multiple Delius matches found"
+    val exceptionsToRetryOn = listOf(HttpClientErrorException::class, HttpServerErrorException::class)
```

Comment:
No list was given(not part of the story but better this way) so added the usual ones as suggested.